signac/version.py

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2019 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+
+
+__version__ = "1.2.0"

```

Comment:
We should explicitly expose `__version__` here.
```suggestion
__version__ = "1.2.0"

__all__ = ['__version__']
```